### PR TITLE
fix TimeoutSeconds Default second.

### PR DESCRIPTION
**Pull request by llussy**
fix TimeoutSeconds Default second. Default TimeoutSeconds is 1s.

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -2225,7 +2225,7 @@ type Probe struct {
 	// +optional
 	TimeoutSeconds int32 `json:"timeoutSeconds,omitempty" protobuf:"varint,3,opt,name=timeoutSeconds"`
 	// How often (in seconds) to perform the probe.
-	// Default to 10 seconds. Minimum value is 1.
+	// Defaults to 1 second. Minimum value is 1.
```
Comment (aojea) [inline staging/src/k8s.io/api/core/v1/types.go:2228]: I think that you are confusing the field, this comment is about PeriodSeconds, the comment goes before the field 😄 

https://github.com/kubernetes/kubernetes/blob/3e36dd62c66c3d2b1a2cedb019b00e22e79b7c38/pkg/api/v1/defaults.go#L162-L174

**Comment by aojea** [comment]
/close

Thanks but I think that the docs are correct

File: staging/src/k8s.io/api/core/v1/types.go (hunk 1 above)
Comment (llussy) [reply staging/src/k8s.io/api/core/v1/types.go:2228]: Sorry , my fault.

Comment (aojea) [reply staging/src/k8s.io/api/core/v1/types.go:2228]: no worries, I had to look twice too ;)

**Pull request closed without merging**